File: app/src/main/java/com/github/batulovandrey/unofficialurbandictionary/presenter/PopularWordsPresenter.java
package com.github.batulovandrey.unofficialurbandictionary.presenter;

import java.util.ArrayList;

/**
 * @author Andrey Batulov on 31/10/2017
 */

public interface PopularWordsPresenter {

    ArrayList<String> getAlphabet();

    ArrayList<String> getWords(String letter);
}

File: app/src/main/java/com/github/batulovandrey/unofficialurbandictionary/presenter/PopularWordsPresenter.kt
package com.github.batulovandrey.unofficialurbandictionary.presenter

import java.util.*

/**
 * @author Andrey Batulov on 22/12/2017
 */

interface PopularWordsPresenter {

    val alphabet: ArrayList<String>

    fun getWords(letter: String): ArrayList<String>
}